### PR TITLE
Improved readme for better explanation 

### LINKED ISSUE #10: Improve readme
We can improve our readme to be more clear:

- The step "Now you can run travis" is not clear that you have to add the line to travis.yml

### PR DESCRIPTION
**Pull request by AvdLee**
- [x] Added extra assets
- [x] Added more notes on the features this framework adds. More answering "What's in it for me?"
- [x] Improved explanation for the CI script

See a live demo [here](https://github.com/WeTransfer/WeTransfer-iOS-CI/tree/feature/improved-readme)

Fixes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+[![Build Status](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI.svg?branch=master)](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI)
+
 Containing all the shared CI logic for WeTransfer repositories
 
-# Features
-- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow unwrapping for example
+# Why should I use it?
+What's in it for me? Well, quite a lot! With low effort to add it to your project.
+
+- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow `unwrapping optionals` for example
```
Comment (PimCoumans) [inline README.md]: Don’t you mean `force unwrapping`?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+
+### Custom linting
+These warnings are posted inline inside the PR, helping you to solve them easily.
+
+  - Check for `final class` usage
+  - `override` methods without adding logic
+  -  Suggest `weak` over `unowned`
+  -  Suggest `// MARK:` usage for large files
+
+_A comment example. Note that `WeTransferBot` will be replaced by your own bot. More info can be found here: [Getting started with Danger](http://danger.systems/guides/getting_started.html)._
```
Comment (PimCoumans) [inline README.md]: What do you mean with ‘a comment example’?

Comment (PimCoumans) [reply README.md]: A **common** example?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
-# Features
-- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow unwrapping for example
+# Why should I use it?
+What's in it for me? Well, quite a lot! With low effort to add it to your project.
+
+- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow `unwrapping optionals` for example
 - Integrate [Danger](http://danger.systems/) to automatically improve PR reviews
 
+## Danger features
+Following are a list of features which are posted in a comment on PRs based on the submitted files.
```
Comment (Boris-Em) [inline README.md]: Nit: This should be "Following is a list of features..." I believe. There is only one list.

File: README.md (hunk 2 above)
Comment (AvdLee) [reply README.md]: Haha, nope! It's a comment in that image and this text is the caption. I've updated it to be more describing.

**Comment by AvdLee** [comment]
@PimCoumans updated!

**Pull request merged**